src/components/common/footer/missionStatement.css

```diff
@@ -0,0 +1,8 @@
+footer{
```

Comment:
Delete this whole CSS file, you already adjusted the main footer CSS file with the same changes.